src/Bundle/test/config/sylius/grids/book_by_english_authors.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+use App\Entity\Book;
+use App\Entity\Nationality;
+use Sylius\Bundle\GridBundle\Config\Builder\Field\Field;
+use Sylius\Bundle\GridBundle\Config\Builder\Filter\Filter;
+use Sylius\Bundle\GridBundle\Config\Builder\GridBuilder;
+use Sylius\Bundle\GridBundle\Config\Builder\GridConfig;
+
+return static function (GridConfig $grid) {
+    $grid->addGrid(GridBuilder::create('app_book_by_english_authors', Book::class)
+        ->setRepositoryMethod(["expr:service('app.english_books_query_builder')", 'create'])
+        ->addFilter(Filter::create('title', 'string'))
+        ->addFilter(Filter::create('author', 'entity')
+            ->setFormOptions([
+                'class' => Nationality::class,
+            ])
+        )
+        ->addFilter(Filter::create('nationality', 'entity')
+            ->setOptions([
+                'fields' => ['author.nationality'],
+            ])
+            ->setFormOptions([
+                'class' => Nationality::class,
+            ])
+        )
+        ->orderBy('title', 'asc')
+        ->addField(Field::create('title', 'string')
+            ->setLabel('Title')
+            ->setSortable(true)
+        )
+        ->addField(Field::create('author', 'string')
+            ->setLabel('Author')
+            ->setPath('author.name')
+            ->setSortable(true, 'author.name')
+        )
+        ->addField(Field::create('nationality', 'string')
+            ->setLabel('Nationality')
+            ->setPath('na.name')
+            ->setSortable(true, 'na.name')
+        )
+        ->setLimits([10, 5, 15])
```

Comment:
As far as I see, we do not configure any actions in these test configurations. Maybe it would be worth giving them a try? Just to be sure :)

Reply:
@Zales0123 Do we think it could be enough to add actions without testing it via end-to-end tests?
I think it could validate the grid configuration.